### PR TITLE
Modifications in placement_cell

### PR DESCRIPTION
**Pull request by gautamyadavs**

### CODE REVIEW

**Review by arshadsid — COMMENTED** [review]
You can make changes in your repo and make the commits when some version of the final product to a major level is covered. And not for every url added....

**Comment by gautamyadavs** [comment]
@arshadsid, I was just sharing my work with my team members so that @arpitjainds doesn't have to repeat the same things otherwise how are we supposed to work together -_-

**Comment by arshadsid** [comment]
There is thing called fork, using which you dint have ti commit to this repo every single time. Google it.

**Comment by Samy-33** [comment]
@arshadsid All the commits made to the branch of this PR will be visible here, so every time he pushes the changes, it will be shown here. It will continue until the RP is closed or merged.

**Comment by arshadsid** [comment]
What I am saying is they can make commits after substantial amount is done and till then use their forked repo.

**Comment by Samy-33** [comment]
You mean Pull Request.

**Comment by arshadsid** [comment]
Yea Pull Request.

File: FusionIIIT/applications/globals/models.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ class ExtraInfo(models.Model):
     sex = models.CharField(max_length=2, choices=Constants.SEX_CHOICES, default='M')
     age = models.IntegerField(default=18)
     address = models.TextField(max_length=1000, default="")
-    phone_no = models.IntegerField(default=15)
+    phone_no = models.BigIntegerField()
```
Comment (Samy-33) [inline FusionIIIT/applications/globals/models.py:44]: What was the requirement?

File: FusionIIIT/applications/placement_cell/models.py [hunk 2]
```diff
@@ -290,11 +291,11 @@ class StudentPlacement(models.Model):
...
-                                   default='', null=True, blank=True)
+                                   default='NOT PLACED')
     placement_date = models.DateField(_("Date"), default=datetime.date.today, null=True,
                                       blank=True)
     package = models.DecimalField(default='', decimal_places=2, max_digits=5, null=True,
                                   blank=True)
 
     def __str__(self):
-        return self.unique_id.id
+        return self.unique_id.id.id
```
Comment (Samy-33) [inline FusionIIIT/applications/placement_cell/models.py:301]: It is better to return something more readable in `__str__` like name than an integer.

File: FusionIIIT/applications/placement_cell/urls.py [hunk 3]
```diff
@@ -6,6 +6,7 @@
 
 urlpatterns = [
 
-    url(r'^', views.placement, name='placement'),
+    url(r'^$', views.placement, name='placement'),
+    url(r'^profile/(?P<username>[a-zA-Z0-9]{1,20})/$', views.profile, name="profile"),
```
Comment (Samy-33) [inline FusionIIIT/applications/placement_cell/urls.py]: A username can have  a `.` also.

File: FusionIIIT/templates/placementModule/achievements.html [hunk 4]
```diff
@@ -10,7 +10,7 @@
         <div class="ui vertical segment">
```
Comment (Samy-33) [inline FusionIIIT/templates/placementModule/achievements.html:10]: @kmunshi97 the changes in templates. Review them.

File: FusionIIIT/applications/globals/models.py (hunk 1 above)
Comment (gautamyadavs) [reply FusionIIIT/applications/globals/models.py:44]: IntegerField have values from -2147483648 to 2147483647 so I can't store any mobile number like 9479309117

File: FusionIIIT/applications/placement_cell/models.py (hunk 2 above)
Comment (gautamyadavs) [reply FusionIIIT/applications/placement_cell/models.py:301]: It's not an integer try it yourself. It's returning id(roll no) and username both.

File: FusionIIIT/applications/placement_cell/urls.py (hunk 3 above)
Comment (gautamyadavs) [reply FusionIIIT/applications/placement_cell/urls.py]: Done

**Comment by arshadsid** [comment]
@gautamyadavs build is failing check for isort and flake8

**Comment by gautamyadavs** [comment]
I already did that

**Comment by arshadsid** [comment]
Waiting fo @kmunshi97 review

**Pull request merged**